src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -207,7 +209,8 @@ public void launch(final SlaveComputer computer, final TaskListener listener) th
             SWbemServices services = WMI.connect(session, name);
 
 
-            String path = computer.getNode().getRemoteFS();
+            Slave node = computer.getNode();
+            String path = node != null ? node.getRemoteFS() : "";
```

Comment:
Specially review this. If node is null, I am not so sure what the behaviour should be. Maybe throw an error better than what's in the PR ATM?